doc/about.rst

```diff
@@ -220,7 +220,7 @@ The 2013 Paris international sprint
    :target: https://www.tinyclues.com/
 
 
-.. |afpy| image:: https://www.afpy.org/logo.png
+.. |afpy| image:: https://www.afpy.org/static/images/logo.svg
```

Comment:
Aww, great, I couldn't find the right path to the svg logo..

Reply:
I got it thanks to you and the issue you submitted to afpy :-).